pluma/pluma-utils.c

```diff
@@ -846,7 +849,9 @@ pluma_utils_replace_home_dir_with_tilde (const gchar *uri)
 guint
 pluma_utils_get_current_workspace (GdkScreen *screen)
 {
-#ifdef GDK_WINDOWING_X11
+#ifdef HAVE_X11
+if (GDK_IS_X11_DISPLAY (gdk_display_get_default ()))
+{
```

Comment:
`if (GDK_IS_X11_DISPLAY (gdk_display_get_default ()))`

and next lines must have new indentation